documentation/src/main/markdown/downloadingandinstalling/selfupdatingdetect.md

```diff
@@ -1,61 +1,61 @@
 
-If the [company_name] [solution_name] URL of the [company_name] [solution_name] .jar file to download and run has been hardcoded via [company_name] [solution_name] property `DETECT_SOURCE` environment variable or the [company_name] [solution_name] version set by the `DETECT_LATEST_RELEASE_VERSION` or `DETECT_VERSION_KEY` variables, self update will not occur. These are optional System environment properties used by Detect upgrade scripts.
+If the [detect_product_short] URL of the [detect_product_short] .jar file to download and run has been hardcoded via [detect_product_short] property `DETECT_SOURCE` environment variable or the [detect_product_short] version set by the `DETECT_LATEST_RELEASE_VERSION` or `DETECT_VERSION_KEY` variables, self update will not occur. These are optional System environment properties used by [detect_product_short] upgrade scripts.
 
-If the [blackduck_product_name] “Internally Hosted” option has been selected and a [company_name] [solution_name] download location has not been provided, the feature will not be enabled.
+If the [bd_product_short] “Internally Hosted” option has been selected and a [detect_product_short] download location has not been provided, the feature will not be enabled.
 
-For further [blackduck_product_name] configuration information, refer to the documentation provided under the topic:
-<xref href="DetectLocation.dita" scope="peer"> Hosting location for Synopsys Detect.
+For further [bd_product_short] configuration information, refer to the documentation provided under the topic:
+<xref href="DetectLocation.dita" scope="peer"> Hosting location for [detect_product_short].
 <data name="facets" value="pubname=bd-hub"/>
 
-## [company_name] [solution_name] log examples for self update
+## [detect_product_short] log examples for self update
 
 Downgrade to prior version blocked:  
 
 ``` 
-2023-05-05 12:20:57 EDT INFO  \[main] - Detect-Self-Updater:  Checking https://test1.synopsys‎ .com/api/tools/detect API for centrally managed Detect version to download to /Users/testuser/tmp.   
+2023-05-05 12:20:57 EDT INFO  \[main] - Detect-Self-Updater:  Checking https://test1.blackduck‎ .com/api/tools/detect API for centrally managed Detect version to download to /Users/testuser/tmp.   
 
 2023-05-05 12:21:03 EDT WARN  \[main] - Detect-Self-Updater:  The Detect version 8.7.0 mapped at Black Duck server is not eligible for downgrade as it lacks the self-update feature. The self-update feature is available from 8.9.0 onwards.
 ```
 
 Update to version allowed (8.9.0+):   
 
 ```
-2023-05-05 12:33:52 EDT INFO  \[main] - Detect-Self-Updater:  Checking https://test1.synopsys‎ .com/api/tools/detect API for centrally managed Detect version to download to /Users/testuser/tmp.  
+2023-05-05 12:33:52 EDT INFO  \[main] - Detect-Self-Updater:  Checking https://test1.blackduck‎ .com/api/tools/detect API for centrally managed Detect version to download to /Users/testuser/tmp.  
```

Comment:
Is this space deliberate?

Reply:
Ya, to make the link is inactive. (So nobody tries using the sample URL.) I'll have to check again as the invisible space may have been replaced with an actual space by some editing.